File: crates/fixtures/sample.go
package fixtures

import "fmt"

// Helper doubles a value.
func Helper(x int) int {
return x * 2
}

// Process sums items.
func Process(items []int) int {
// TODO: handle empty slice
total := 0
for _, item := range items {
total += Helper(item)
}
return total
}

File: crates/fixtures/sample.java
package fixtures;

import java.util.List;

public class Sample {
    /**
     * Helper doubles a value.
     */
    public static int helper(int x) {
        return x * 2;
    }

    public static int process(List<Integer> items) {
        // TODO: add null check
        int total = 0;
        for (int item : items) {
            total += helper(item);
        }
        return total;
    }
}
